Sources/GDPR/GDPRManager.swift

```diff
@@ -73,16 +87,20 @@ public class GDPRManager {
 
     public func setServices(services: [Service]) {
```

Comment:
`public func setServices(services: Set<Service>) {`
If you use a set you don't need to assert for duplicates.

Reply:
but we want to control the order of the services